src/test/scala/org/ergoplatform/nodeView/state/UtxoStateSpecification.scala

```diff
@@ -76,7 +76,8 @@ class UtxoStateSpecification extends ErgoPropertyTest with ErgoTransactionGenera
         height = height,
         parentId = us.stateContext.lastHeaderOpt.map(_.id).getOrElse(Header.GenesisParentId))
       val adProofs = ADProofs(realHeader.id, adProofBytes)
-      val fb = ErgoFullBlock(realHeader, BlockTransactions(realHeader.id, txs), genExtension(realHeader, us.stateContext), Some(adProofs))
+      val bt = BlockTransactions(realHeader.id, 1: Byte, txs)
```

Comment:
```suggestion
      val bt = BlockTransactions(realHeader.id, Header.InitialVersion, txs)
```
This property can also be tested cross-version.